### PR TITLE
Update Blog Post “2015/02/2015-02-02-agency-perspectives-on-personas-use-development-and-challenges”

### PR DESCRIPTION
**Pull request by bonnieAcameron**
Automatically generated by Netlify CMS

### CODE REVIEW

**Review by afeijoo — DISMISSED** [review]

File: content/news/2015/02/2015-02-02-agency-perspectives-on-personas-use-development-and-challenges.md [hunk 1]
```diff
@@ -2,65 +2,74 @@
...
+# Controls how this page appears across the site
+# 0 -- hidden
+# 1 -- visible
+weight: 1
 ---
-
 Personas are tools your agency can use to learn about your end users and drive decisions. Personas are so useful because they serve as a communication tool for your team. You can keep these personas in mind to guide any work that your agency performs.
 
-Let&#8217;s delve a bit deeper into [personas]({{< ref "2015-01-09-personas-101.md" >}}) and review two examples from the federal community. Below, we have personas from the Department of Human and Health Services (HHS) and the United States Department of Agriculture (USDA).
+Let&#8217;s delve a bit deeper into \[personas]({{< ref "2015-01-09-personas-101.md" >}}) and review two examples from the federal community. The Department of Human and Health Services (HHS) and the United States Department of Agriculture (USDA) both
```
Comment (mejiaj) [inline content/news/2015/02/2015-02-02-agency-perspectives-on-personas-use-development-and-challenges.md:28]: Removed HTML code for apostrophe and slash before the link (it was causing it to render incorrectly).

```suggestion
Let's delve a bit deeper into [personas]({{< ref "2015-01-09-personas-101.md" >}}) and review two examples from the federal community. The Department of Human and Health Services (HHS) and the United States Department of Agriculture (USDA) both 
```

---

Is it supposed to end on `both`?
image

**Review by mejiaj — COMMENTED** [review]
@bonnieAcameron looks good overall, just a comment on one of the paragraphs. It seems like  the link is broken in [preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.app.cloud.gov/preview/gsa/digitalgov.gov/cms/news/2015/02/2015-02-02-agency-perspectives-on-personas-use-development-and-challenges/2015/02/02/agency-perspectives-on-personas-use-development-and-challenges/) and the paragraph suddenly ends.

File: content/news/2015/02/2015-02-02-agency-perspectives-on-personas-use-development-and-challenges.md (hunk 1 above)
Comment (bonnieAcameron) [reply content/news/2015/02/2015-02-02-agency-perspectives-on-personas-use-development-and-challenges.md:28]: You're right! It isn't, good catch.

**Comment by bonnieAcameron** [comment]
@mejiaj is this ready to be merged? Let me know if I missed something!

**Comment by mejiaj** [comment]
@bonnieAcameron still seeing the issues I commented on. Perhaps you forgot to push your latest changes?

**Review by mejiaj — APPROVED** [review]

**Pull request merged**